### PR TITLE
Update test targets ruby versions

### PR DESCRIPTION
**Pull request by yui-knk**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,11 @@ language: ruby
...
 rvm:
-  # - 1.8.7
-  # - 1.9.2
   - 1.9.3
   - 2.0.0
-  - 2.1
-  - 2.2
+  - 2.1.8
+  - 2.2.4
+  - 2.3.0
```
Comment (alto) [inline .travis.yml]: I would like to keep using the short versions (like `2.2`), making sure that the latest minor version available will be picked.

**Comment by alto** [comment]
@yui-knk Thanks for your pull request. Please change according to my comment and I'm happy to merge.

**Comment by yui-knk** [comment]
I updated, can you check it ? :smile_cat:

**Comment by yui-knk** [comment]
> the latest minor version available will be picked

Oh, Travis CI does not pick up minor latest version...  For example https://travis-ci.org/aasm/aasm/jobs/108457677 used `ruby 2.2.0p0`.

**Comment by alto** [comment]
@yui-knk I manually merged your PR, and by this told Travis CI to ignore failing builds for Ruby 2.3 which is not yet provided. 

Thanks for your help!

**Pull request merged**